src/Row.js

```diff
@@ -104,15 +104,14 @@ var Row = React.createClass({
 
   getCellValue(key: number | string): any {
     var val;
-    if(key === 'select-row'){
+    if(key === 'select-row') {
       return this.props.isSelected;
     } else if (typeof this.props.row.get === 'function') {
       val = this.props.row.get(key);
+    } else {
+      val = this.props.row[key];
     }
-    else {
-      var val = this.props.row[key];
-    }
-    return !val ? '' : val;
+    return val;
```

Comment:
@malonecj  as we discussed it on slack , returning the underlying value here, so that if the value is false is not replaced with an empty string.